package.json

```diff
@@ -37,17 +37,18 @@
 		"jquery": "^1.7 || ^2.0"
 	},
 	"devDependencies": {
-		"commitplease": "^2.2.3",
-		"grunt": "^0.4.4",
-		"grunt-contrib-compress": "^0.7",
-		"grunt-contrib-concat": "^0.3",
-		"grunt-contrib-copy": "^0.5",
-		"grunt-contrib-jshint": "^0.11.3",
-		"grunt-contrib-qunit": "^0.4",
-		"grunt-contrib-uglify": "^0.4",
-		"grunt-contrib-watch": "^0.6",
-		"grunt-jscs": "^2.2",
-		"grunt-text-replace": "^0.3.11"
+		"commitplease": "2.3.1",
+		"grunt": "1.0.1",
+		"grunt-contrib-compress": "1.2.0",
```

Comment:
I followed the convention used in most of `jQuery` projects (`QUnit`, `jquery`, `jquery-ui` ...).

Here is a quote from one of the core developpers of `jQuery`:
>Most of jQuery repos went in the opposite direction. Semver is an idea but reality is that from time to time a package breaks and then we have a problem. IMO it's better to just pin versions and update from time to time. Note that using non-latest version of a dependency isn't a bad thing; maybe they introduced a new feature we don't use, maybe a bug fix that doesn't affect us that much. It's good to not lock down to something ancient but it doesn't mean constantly living on the bleeding edge is an adequate answer either

But if you want, I can leverage all the dev-dependencies.